components/UI/modal/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from 'components/UI/modal/modal.module.scss';
+import Backdrop from 'components/UI/backdrop';
+
+const Modal = ({ show, modalClosed, children }) => (
+  <>
+    <Backdrop show={show} clicked={modalClosed} />
+    <div
+      className={classes.Modal}
```

Comment:
are we following a convention here as I observe camelCase for `crossIcon` and `Modal`  here